website/docs/r/cluster.html.markdown

```diff
@@ -298,6 +298,7 @@ But in order to explicitly change `provider_instance_size_name` comment the `lif
     - AZURE - Microsoft Azure
 
 * `provider_disk_iops` - (Optional - AWS Only) The maximum input/output operations per second (IOPS) the system can perform. The possible values depend on the selected `provider_instance_size_name` and `disk_size_gb`.  This setting requires that `provider_instance_size_name` to be M30 or greater and cannot be used with clusters with local NVMe SSDs.  The default value for `provider_disk_iops` is the same as the cluster tier's Standard IOPS value, as viewable in the Atlas console.  It is used in cases where a higher number of IOPS is needed and possible.  If a value is submitted that is lower or equal to the default IOPS value for the cluster tier Atlas ignores the requested value and uses the default.  More details available under the providerSettings.diskIOPS parameter: [MongoDB API Clusters](https://docs.atlas.mongodb.com/reference/api/clusters-create-one/)
+  * The default IOPS value for the AWS clusters is now 3000 upto 1TB standard EBS storage. You do not need to configure IOPS for the Standard EBS configuration.
```

Comment:
We should not add all of this - as noted above they should view in the Atlas console as we can't keep up with changes in a human generated fashion in docs right now.   
If you want to add "You do not need to configure IOPS for a STANDARD disk configuration but only for a PROVISIONED configuration. " I think that would make sense.